server/lib/realtime_web/channels/realtime_channel.ex

```diff
@@ -3,6 +3,7 @@ defmodule RealtimeWeb.RealtimeChannel do
   require Logger, warn: false
 
   def join("realtime:" <> _topic, _payload, socket) do
+    send(self(), :after_join)
```

Comment:
there is no need to make another message send, you can call the `track_channel` here

Reply:
yep, good call!